src/forms/File/_file.scss

```diff
@@ -1,15 +1,17 @@
 /*
  * File
  * Styles for File upload inputs.
+ *
+ *       <div class="form-file">
+ *           <label class="form-label" for="file">File</label>
+ *           <input class="form-input" id="file" name="file" type="file" value="">
+ *       </div>
  */
 
-.file-input {
-	padding: rhythm(0.5);
-	background-color: color('canvas');
-	border: 1px dashed color('mid');
-	text-align: center;
-}
-
-.file-input__preview {
-	
+.form-file > .form-input {
```

Comment:
So this is the main point of disagreement for me. 

I feel like this should be:

```
.form-file
.form-file__input {
width: 100%;
    padding: rhythm(0.25);
    background-color: color(canvas);
    border: 1px dashed color(base);
    text-align: center;
}

```

and then markup would be:

```
<div class="form-file">
      <label class="form-label" for="file">File</label>
      <input class="form-input form-file__input" id="file" name="file" type="file" value="">
</div>
```

Reply:
Yeah, happy with that
I'll see if I can amend and push up

Reply:
@darylthornhill @liamco

Reply:
I am happy with the above if we decide to do this route, HOWEVER.....

As we were talking about it in slack we had `form-input--email` as an example.
Should we be doing `form-input--file` as its a modifier of input?

Reply:
Ah, yes, modifiers for text type inputs. 

I kinda class file as separate from the text type though, but I can see why we _could_ add it as a modifier? It might depend on a case by case basis

Reply:
Let's go the route of this being a completely separate block with the input as an element as per your example. 
I feel like if there's a preview and other bits that the block is dependent on then sure it can become its own component, so yeah it works for me!